### PR TITLE
Add SSH documentation to the website.

### PR DESCRIPTION
**Pull request by rgrandl**

### CODE REVIEW

File: website/docs.md [hunk 1]
```diff
@@ -3089,6 +3091,134 @@ slowly shifts traffic from old versions of the application to the new version of
...
+$ weaver ssh deploy weaver.toml
+```
+
+When `weaver ssh deploy` terminates (e.g., when you press `ctrl+c`), the
+application is destroyed and all processes are terminated.
+
+## Logging
+
+`weaver ssh logs` logs to stdout. Refer to `weaver ssh logs --help` for a full
+explanation of the query language, along with many more examples.
```
Comment (mwhittaker) [inline website/docs.md:3150]:
```suggestion
`weaver ssh logs` logs to stdout. Refer to `weaver ssh logs --help` for details.
```

**Review by mwhittaker — APPROVED** [review]
Nice, thanks Robert!

File: website/docs.md [hunk 2]
```diff
@@ -3089,6 +3091,134 @@ slowly shifts traffic from old versions of the application to the new version of
...
+locations = "./ssh_locations.txt"
+```
+
+The `[serviceweaver]` section of the config file specifies the compiled Service
+Weaver binary. The `[ssh]` section contains the set of machines where your
+application should be deployed, and per listener configuration. The set of
+machines are specified as follows:
+
+```txt
+// ssh_locations.txt
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply website/docs.md:3130]: Yeah, I don't think so. Reworded.

File: website/docs.md (hunk 1 above)
Comment (rgrandl) [reply website/docs.md:3150]: Done

**Review by rgrandl — COMMENTED** [review]
Thanks Michael!

**Pull request merged**